config.yaml

```diff
@@ -1,281 +1,432 @@
+  - jschanck
+  - Martyrshot
+  - praveksharma
+  - SWilson4
+  - vsoftco
+# liboqs CODEOWNERS
+# https://github.com/open-quantum-safe/liboqs/blob/main/.github/CODEOWNERS
+- name: liboqs-codeowners
   maintainers:
   - dstebila
-- name: minute-takers
-  maintainers:
+  members:
+  - alexrow
+  - baentsch
+  - bhess
+  - crockeea
+  - jschanck
   - SWilson4
-- name: openssh-committers
+
+# oqs-provider Maintainers
+# https://github.com/open-quantum-safe/oqs-provider/blob/main/GOVERNANCE.md#maintainers-1
+- name: oqs-provider-maintainers
   maintainers:
-  - dstebila
+  - baentsch
+# oqs-provider Committers
+# https://github.com/open-quantum-safe/oqs-provider/blob/main/GOVERNANCE.md#committers-1
+- name: oqs-provider-committers
+  maintainers:
+  - baentsch
   members:
-  - geedo0
+  - bhess
+  - christianpaquin
+  - thb-sb
+# oqs-provider CODEOWNERS
+# https://github.com/open-quantum-safe/oqs-provider/blob/main/.github/CODEOWNERS
 - name: oqs-provider-codeowners
   maintainers:
   - baentsch
   members:
+  - alexrow
   - bhess
   - feventura
   - iyanmv
+  - jplomas
   - thb-sb
-- name: oqs-provider-committers
+
+# boringssl Maintainers
+# TODO: provide "source of truth"
+- name: boringssl-maintainers
   maintainers:
-  - thb-sb
-- name: read
+  - pi-314159
+
+# openssh Release Managers
+# TODO: provide "source of truth"
+- name: openssh-release-managers
   maintainers:
   - dstebila
   members:
-  - jplomas
-- name: rust
+  - baentsch
+  - geedo0
+  - praveksharma
+
+# liboqs-cpp Maintainers
+# TODO: provide "source of truth"
+- name: liboqs-cpp-maintainers
   maintainers:
-  - thomwiggers
-- name: triage
+  - vsoftco
+# liboqs-cpp Release Managers
+# TODO: provide "source of truth"
+- name: liboqs-cpp-release-managers
   maintainers:
-  - dstebila
-  members:
-  - planetf1
-  - ajbozarth
-  - geedo0
-- name: trail-of-bits
+  - vsoftco
+
+# liboqs-go Maintainers
+# TODO: provide "source of truth"
+- name: liboqs-go-maintainers
   maintainers:
-  - SWilson4
-  members:
-  - fcasal
-  - tob-scott-a
-- name: tsc
+  - vsoftco
+# liboqs-go Release Managers
+# TODO: provide "source of truth"
+- name: liboqs-go-release-managers
+  maintainers:
+  - vsoftco
+
+# liboqs-python Maintainers
+# TODO: provide "source of truth"
+- name: liboqs-python-maintainers
+  maintainers:
+  - vsoftco
+# liboqs-python Release Managers
+# TODO: provide "source of truth"
+- name: liboqs-python-release-managers
+  maintainers:
+  - vsoftco
+
+# liboqs-java Maintainers
+# TODO: provide "source of truth"
+- name: liboqs-java-maintainers
   maintainers:
-  - baentsch
   - dstebila
   members:
-  - ashman-p
-  - bhess
-  - brian-jarvis-aws
-  - christianpaquin
-  - thb-sb
-  - vsoftco
-- name: www
+  - jimouris
+
+# liboqs-rust Maintainers
+# TODO: provide "source of truth"
+- name: liboqs-rust-maintainers
   maintainers:
-  - crockeea
+  - thomwiggers
+# liboqs-rust Release Managers
+# TODO: provide "source of truth"
+- name: liboqs-rust-release-managers
```

Comment:
By renaming this team, the Crates.io ownership got invalidated. Fortunately, I'm still a direct owner of the `oqs` crates on `crates.io`.

See https://crates.io/teams/github:open-quantum-safe:rust

Reply:
Thanks for letting us know, Thom. Is it easy to change the team reference in crates.io?  If not we can change the team name back here.  Either way @SWilson4 can we create a comment in the file noting that the team name is connected to permissions on crates.io?

Reply:
Yes—I'll put up a PR to do that. If necessary I'll also change the team name back to "rust"; just let me know @thomwiggers.

Reply:
See https://github.com/open-quantum-safe/tsc/pull/85.